### ISSUE TITLE
Preview crashes after click on link

### ISSUE THREAD

**Issue by AKhranovskiy** [joaompinto/asciidoctor-vscode#10]
**STR:**
1. Create document with TOC or image thumbnail
2. Open preview
3. Click on link

**Expected:**
Follow a link

**Given:**
Preview document closes, error popups: "Unable to open '/Preview: 'filename.adoc'': Cannot read property 'document' of undefined."
Preview stops working till restart of an application.

VScode version: Version 1.10.1, 653f8733dd5a5c43d66d7168b4701f94d72b62e5
plugin version: 0.3.0

**Comment by joaompinto** [comment]
Can you please provide me an example .adoc ? 
Thanks

**Comment by AKhranovskiy** [comment]
```
Sample adoc with crash
=======================
:toc:
:numbered:

== Header
== Header
=== Header
```

Open preview, click on item in TOC.
Result: crash.

**Comment by nkrauth** [comment]
I'm still facing the issue with version 0.3.2 of the plugin. VScode Version is 1.10.2. 

I happens every time when clicking on a link but also randomly when working on documents.

**Comment by nkrauth** [comment]
Another scenario that results in the error popup: "Unable to open '/Preview: 'filename.adoc'': Cannot read property 'document' of undefined.":

1. open the sample document provided above in VScode and display the adoc-Preview
2. open a copy of the same document in the same instance of VScode and display the adoc-Preview
3. switch back to the tab of the first document

**Comment by joaompinto** [comment]
Thanks for reporting. Working on it right now.

**Comment by joaompinto** [comment]
The links in the TOC is now fixed on 0.3.3, I have open issue #13 for the multiple tabs crash, this is a different issue. The TOC-alike crash would also happen on any kind of link to local document sections.
